pyproject.toml

```diff
@@ -93,6 +93,8 @@ datasets = [
     "pyvista>=0.34.2",
     # radiant-mlhub 0.3+ required for newer tqdm support required by lightning
     "radiant-mlhub>=0.3",
+    # source-cooperative 
+    "source-cooperative>=beta",
```

Comment:
Tests are failing because this isn't a real Python library

Reply:
Ohh yes! should be cooperative, and version is 0.1.12. Fixing right away